### DISCUSSION TITLE
Strange folding problem with long To: line

### DISCUSSION THREAD

**Discussion by pmaubertel** [PHPMailer/PHPMailer#2653] [Q&A]
Hi,
I'm having a strange folding issue with long "To:" lines into a message 
with many recipients (>30). 
I'm using PHPMailer 6.6.0 on a php 7.3.3 Linux server with the "mail" function of PHP.

The long "To:" line gets folded in chunks of 998 chars as requested by the RFC
but the last line is interrupted suddenly for no reason. 
This incomplete line produces a 550 error :
    5.7.1 Missing 'From' header (in reply to end of DATA command)

Is there a configuration option I'm missing ?
Here after the message after hiding of sensitive information.

![Mail_header](https://user-images.githubusercontent.com/42812899/159177680-57131553-cc8d-4481-ab55-985c15e9995d.png)

**Comment by Synchro** [comment]
If you set `SMTPDebug = 2`, even when using `mail()`, you should see a copy of the message as generated by PHPMailer. It could be that the mail function is truncating the header – perhaps compare with sending via SMTP?

**Reply by pmaubertel** [reply]
Thanks for the suggestion, I've setup the debug option but I'm obliged to wait for the problem to re-appear.
I'll keep you informed here.
